.github/workflows/cicd.yml

```diff
@@ -578,6 +578,16 @@ jobs:
       environment: ebpf_cicd_perf_ws2022
       configurations: '["Release"]'
 
+  netperf:
+    if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
+    uses: ./.github/workflows/netperf.yml
+    with:
+      sha: ${{ github.sha }}
+      ref: ${{ github.ref }}
+      pull_request: ${{ github.event.pull_request.number }}
```

Comment:
does this field contain any data if the trigger event is schedule or dispatch?

Reply:
No, it will be empty. On the other hand we eventually want perf to run on each PR. This is blocked on scale in the microsoft/netperf repo.